extern crate failure;

use failure::Error;
use std::io::{Read, Write};

pub fn answers(input: &mut Read, output: &mut Write) -> Result<(), Error> {
    Ok(())
}
